crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,58 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
 Run flake application from the requested package (or "installable").
 If not provided `flox` will prompt for you to select from the list of known packages.
+`flox run` uses `nix run` under the hood to execute the so-called installable. Which makes it possible to run 
+packages from any Nix flake that exposes the `app` attribute in its outputs.
```

Comment:
I think `nix` will run `packages` and `legacyPackages` as well https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-run.html#flake-output-attributes Saying all of that is probably too detailed

Reply:
Would redirecting people to [nix run](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-run.html) docs be too much?

Reply:
Copying my other comment, my personal opinion is:
Honestly I think if we just give a high-level description that's probably best. So that users don't have to understand flakes, outputs, apps vs packages, etc